wagtail/admin/templates/wagtailadmin/shared/breadcrumbs.html

```diff
@@ -20,6 +20,7 @@
              data-w-breadcrumbs-open-icon-class="icon-breadcrumb-expand"
              data-w-breadcrumbs-opened-content-class="w-max-w-4xl"
              data-w-breadcrumbs-peek-target-value="header"
+             data-w-breadcrumbs-store-value="header-breadcrumbs"
```

Comment:
The identifier will be used as the key. I think we won't need to add the word breadcrumbs to the value also. 

```suggestion
             data-w-breadcrumbs-store-value="header"
```

Also, be sure to check the breadcrumbs used outside of the header are not storing anything. I'm pretty sure they won't be instantiated, but please check. The page chooser modal for example.

Reply:
I checked everywhere in the browser, mainly where the page chooser modal is used and in the code. Breadcrumbs used outside of the header do not store anything.